litellm/utils.py

```diff
@@ -9724,6 +9724,8 @@ def chunk_creator(self, chunk):
                             "completion_tokens"
                         ],
                         total_tokens=anthropic_response_obj["usage"]["total_tokens"],
+                        cache_read_input_tokens=anthropic_response_obj['usage']['cache_read_input_tokens'],
```

Comment:
this is not a guarantee - this block is used for any streaming iterator which returns a GenericStreamingChunk 

You'd also want to update the GenericStreamingChunk type, if you expect this to be a required field

Reply:
> this is not a guarantee - this block is used for any streaming iterator which returns a GenericStreamingChunk
> 
> You'd also want to update the GenericStreamingChunk type, if you expect this to be a required field

@krrishdholakia Would Mistral be one that uses `GenericStreamingChunk`?

Reply:
no - it's used by vertex ai, gemini (google ai studio), bedrock, and anthropic

Reply:
I switched to `.get()`